### ISSUE TITLE
Disable some projects admin fields if there are stores

### ISSUE THREAD

**Issue by julen** [translate/pootle#3475]
The previous projects admin UI disabled some fields when the project had stores in it: file type and tree style. I guess this was done as a safety measure to avoid messing up with existing data, so let's try to bring this back.

**Comment by iafan** [comment]
Doesn't make much sense now, since the 'disabled' flag can be easily cleared by an admin. This makes sense only when we split 'disabled' property into 'disabled'+'hidden', as we discussed recently.

**Comment by julen** [comment]
@iafan I believe you misread, this has nothing to do with the `disabled` field of projects, it's about setting the mentioned drop-down widgets as disabled (`localfiletype` and `treestyle`) when there are stores in the project, as changing these settings can badly affect the health of a project.

**Comment by unho** [comment]
Yes, I think it is a good idea to avoid altering those fields once the Projects are created.

**Comment by iafan** [comment]
@julen, sorry. Yes, this makes sense then
